README.md

```diff
@@ -69,6 +69,15 @@ $ sort-package-json "**/package.json" --check
 # 2 of 5 matched files are not sorted.
 ```
 
+#### `--quiet` flag
```

Comment:
Better split this to anothet PR. It will be easier to merge a refactor  PR.

Reply:
I created #281, but I will need to edit it before reopening to ensure no merge conflicts. I can't do that right now, but hopefully I will have it done before the end of the day